index.html

```diff
@@ -12,10 +17,161 @@
+          data-bs-target="#sidebar"
+          aria-controls="sidebarNavbar"
+        >
+          <img
+            src="./assets/images/burger-bar.png"
+            width="40px"
+            alt="menu bars"
+          />
+        </button>
+        <a class="navbar-brand d-none d-lg-block" href="#">
+          <div
+            class="logo d-flex align-items-center justify-content-center gap-1"
+          >
+            <img
+              src="assets/images/rock_fest_logo.jpg"
+              width="55px"
+              alt="rock fest"
+            />
+            <h5 class="logo-name">
+              rock<span class="logo-name-orange">fest</span>
+            </h5>
+          </div>
+        </a>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mb-2 mb-lg-0"></ul>
+          <a href="#" title="Buy ticket" class="navbar-btn">Buy Ticket</a>
+        </div>
+      </div>
+    </nav>
+
+    <main id="main"></main>
+
+    <section id="partners">
+      <div class="container py-5 px-0">
+        <h1
+          class="partner-section-title section-title text-white text-center fw-bold"
+        >
+          Partner
+        </h1>
+
+        <div
+          class="partners-content d-flex align-items-center justify-content-center flex-wrap gap-4 gap-lg-5 mt-5"
+        >
+          <div class="partner-card">
+            <img src="assets/images/mozilla.png" width="100%" alt="mozilla" />
+          </div>
+          <div class="partner-card">
+            <img src="assets/images/google.png" width="100%" alt="google" />
+          </div>
+          <div class="partner-card">
+            <img src="assets/images/naver.png" width="100%" alt="naver" />
+          </div>
+          <div class="partner-card">
+            <img src="assets/images/kakao1.png" width="100%" alt="kakao" />
+          </div>
+          <div class="partner-card">
+            <img src="assets/images/airbnb.png" width="100%" alt="airbnb" />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer id="footer">
+      <div
+        class="container py-5 px-0 d-flex align-items-center justify-content-between justify-content-sm-between justify-content-md-between justify-content-lg-start gap-lg-5"
+      >
+        <div
+          class="logo d-flex align-items-center justify-content-center gap-1"
+        >
+          <img
+            src="assets/images/rock_fest_logo.jpg"
+            width="55px"
+            alt="rock fest"
+          />
+          <h5 class="logo-name">
+            rock<span class="logo-name-orange">fest</span>
+          </h5>
+        </div>
+        <div
+          class="d-flex align-items-start align-items-sm-start align-items-md-center align-items-lg-center flex-column flex-sm-column flex-md-row flex-lg-row gap-lg-2"
+        >
+          <p>2023 Rockfest by Designesia.</p>
+          <p>Some Rights Reserved.</p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] For the about footer page, the background-color is supposed to be black, not white.
- [ ] Also the parner section is not supposed to be displayed on the about footer page. Kindly take a look at both designs and adjust your design according to the guiding template.

### Your About footer page
![your-about-footer](https://user-images.githubusercontent.com/26597432/217543273-e386f5f3-d1b1-41fe-a6d4-453359beca2d.png)

### Guiding template about footer page
![temp-about-footer](https://user-images.githubusercontent.com/26597432/217543351-4074189f-61a7-4588-bcef-228747fa12e9.png)